segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/operations/MergeSegmentOperation.java

```diff
@@ -47,6 +54,12 @@ public MergeSegmentOperation(long targetSegmentId, long sourceSegmentId) {
         this.streamSegmentOffset = -1;
     }
 
+    public MergeSegmentOperation(long targetSegmentId, long sourceSegmentId, AttributeUpdateCollection attributeUpdates) {
+        this(targetSegmentId, sourceSegmentId);
```

Comment:
I think it's simpler if you make the other constructor invoke this one (with `null` for the last arg), and move the body of the other constructor in here.

Reply:
Done.